docs_user/modules/proc_stopping-openstack-services.adoc

```diff
@@ -183,3 +168,7 @@ for i in {1..3}; do
     fi
 done
 ----
+
+.Verification
+
+* Verify that the control plane services are stopped.
```

Comment:
How do you verify that the control plane services are stopped?

Reply:
It's part of the snippet above, the parts "Checking systemd OpenStack services" and "Checking pacemaker OpenStack services" are doing the verification. It either prints an "OK" or "ERROR" per each service. So the verification here could be that there are no "ERROR" messages from the above snippet.

Reply:
Thank you.
If any services show an ERROR, what would the customer do? Confirm that they did all the necessary prerequisites and try again?

Reply:
They could try again but i think it's likely it would require troubleshooting. I don't think we've ever seen these services fail to stop correctly so i think this is an unlikely spot to hit issues.